sources/packages/web/src/components/students/applicationTracker/Assessment.vue

```diff
@@ -13,11 +14,20 @@
 </template>
 <script lang="ts">
 import ApplicationStatusTrackerBanner from "@/components/students/applicationTracker/generic/ApplicationStatusTrackerBanner.vue";
+import RelatedApplicationChanged from "@/components/students/applicationTracker/RelatedApplicationChanged.vue";
 import { defineComponent } from "vue";
 
 export default defineComponent({
   components: {
     ApplicationStatusTrackerBanner,
+    RelatedApplicationChanged,
+  },
+  props: {
+    showRelatedApplicationChanged: {
+      type: Boolean,
+      required: false,
+      default: false,
+    },
```

Comment:
At the bare minimum please the `Assessment.vue` must receive the `triggerType` value and the component must decide what to display based on trigger type.

I was event thinking if `Assessment.vue` should have it's own API like inProgress, enrolment and completed. But as we only need trigger type I am ok without API.